### ISSUE TITLE
[Bug] Plugin picks up sourcemaps as pages

### ISSUE THREAD

**Issue by jariz** [ematipico/terraform-nextjs-plugin#296]
# Issue

## Expected Behavior
The plugin only looks for `.next/serverless/pages/*.js` files.

## Actual Behavior
The plugin picks up every file in `.next/serverless/pages`, including `*.js.map` files.

![](https://jari.lol/Lm4TWHRP4j.png)

## Steps to Reproduce the Problem
Simply running a build with the latest version of next should suffice.
I do have a few custom settings in my webpack config but none of them have to do with sourcemapping, which as far as I know is enabled by default in nextjs.

## Specifications

- Version: 3.0.0
- Platform: Darwin
- Subsystem: n/a

**Comment by ematipico** [comment]
Hi @jariz , thank you for reporting this. Did you use some kind of plugin? As far as I know Next.js doesn't emit source maps for the server code.

**Comment by jariz** [comment]
Fair enough, I'll try reproducing this from a 'fresh' nextJS instance, will let you know.

**Comment by jariz** [comment]
So I looked into this a bit more, and yes, the cause of this is a plugin after all:
https://github.com/vercel/next-plugins/tree/master/packages/next-source-maps

I guess the sourcemaps aren't really needed for the serverside rendered pages, but it's kind of a hard thing to turn off conditionally from the webpack config.
Also, I don't think it can hurt to make this plugin only look for `*.js` files instead of grabbing all files in the pages dir.
If you agree, then I won't mind opening a PR that implements this.

**Comment by ematipico** [comment]
So we would like to upload also the source maps, which is a nice to have! If you want to give it a shot, please go a ahead! Let me know if you have any questions
